server/server.go

```diff
@@ -1094,7 +1095,7 @@ func (s *Server) campaignLeader() {
 		zap.String("campaign-pd-leader-name", s.Name()),
 		zap.String("purpose", s.member.Leadership.Purpose))
 	if err := s.member.Leadership.Campaign(s.cfg.LeaderLease, s.member.GetLeaderPath(), s.member.MemberValue()); err != nil {
-		log.Error("campaign pd leader meet error", zap.Error(err))
+		log.Error("campaign pd leader meet error", errs.ZapError(errs.ErrCampaignLeader, err))
```

Comment:
Does this is already a RFC error?

Reply:
No. It is a local error caused by lease or etcd.

Reply:
No, we need to wrap the third error to RFC error in `member` package rather than in the server package.  if the error is not returned by a third-party package, just print the log directly or return the error directly.